### PR TITLE
Issue#1

### LINKED ISSUE #1: File Path Updates Needed in Detector & Evaluator Module
Hi,

When running:
`python3 -m evaluator.calc.calc_accuracy
`
I encounter the following error:

FileNotFoundError: [Errno 2] No such file or directory: 'detector/models/t5_hidden/settings.yaml'
Moreover, when executing:

`from evaluator.interpret.integrated_gradient import explain
`
I encounter a similar issue:
FileNotFoundError: [Errno 2] No such file or directory: 'detector/models/t5_hidden/settings.yaml'

I attempted to correct the paths, but it appears some files are missing, such as the "models-small" directory that needs to be loaded in evaluator.interpret.integrated_gradient.explain. It appears that the repository requires updates regarding file paths and names.

Thank you!

### PR DESCRIPTION
**Pull request by haok1402**
fix #1

### CODE REVIEW

File: detector/t5_sentinel/types.py [hunk 1]
```diff
@@ -2,17 +2,25 @@
...
-from pydantic import BaseModel, validator
+from pydantic import BaseModel, validator, ConfigDict
 
 
 class BackboneField(BaseModel):
     name: str
     model_max_length: int
 
-    @validator('name')
+    model_config = ConfigDict(protected_namespaces=())
```
Comment (MarkChenYutian) [inline detector/t5_sentinel/types.py:12]: Why is this added?

File: detector/t5_sentinel/types.py [hunk 2]
```diff
@@ -45,10 +53,10 @@ class TokenizerField(BaseModel):
     truncation: bool
     return_tensors: str
 
-    @validator('return_tensors')
+    @validator("return_tensors")
     def verify_return_tensors(cls, v: str):
-        if v not in ['pt', 'tf', 'np']:
-            raise ValueError(f'Returning tensors with {v} is not supported!')
+        if v not in ["pt", "tf", "np"]:
+            raise ValueError(f"Returning tensors with {v} is not supported!")
```
Comment (MarkChenYutian) [inline detector/t5_sentinel/types.py:59]: Maybe we can remove `tf` and `np` since all other pipelines we built only support `pt` mode.

File: evaluator/data_analysis/plot_token_distribution.py [hunk 3]
```diff
@@ -50,7 +61,8 @@ def filter_dict(counter: dict, keys: list) -> dict:
 
 def merge_keys(*arr_keys) -> list:
```
Comment (MarkChenYutian) [inline evaluator/data_analysis/plot_token_distribution.py:62]: This function seems to be dead (not referenced by anyone else) and we should remove it.

File: detector/t5_sentinel/types.py (hunk 1 above)
Comment (haok1402) [reply detector/t5_sentinel/types.py:12]: this is to silence Pydantic warning with any field starting in `model_` prefix. Otherwise, the field `model_max_length` would cause a warning.
